src/SFA.DAS.AssessorService.Web.Staff/Controllers/CertificateApprovalsController.cs

```diff
@@ -8,47 +10,303 @@
+            var items = Mapper.Map<List<CertificateDetailApprovalViewModel>>(certificates.Items);
+            var certificatesApproved = new CertificateApprovalViewModel
+            {
+                ApprovedCertificates = new PaginatedList<CertificateDetailApprovalViewModel>(items, certificates.TotalRecordCount, certificates.PageIndex, certificates.PageSize)
+            };
+
+            return View(certificatesApproved);
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> Rejected(int? pageIndex)
+        {
+            var certificates = await ApiClient.GetCertificatesToBeApproved(PageSize, pageIndex ?? 1, CertificateStatus.Draft, CertificateStatus.Rejected);
+            var items = Mapper.Map<List<CertificateDetailApprovalViewModel>>(certificates.Items);
+            var certificatesThatAreRejected = new CertificateApprovalViewModel
+            {
+                RejectedCertificates = new PaginatedList<CertificateDetailApprovalViewModel>(items, certificates.TotalRecordCount, certificates.PageIndex, certificates.PageSize)
+            };
+
+            return View(certificatesThatAreRejected);
+        }
+
         [HttpPost(Name = "Approvals")]
         public async Task<IActionResult> Approvals(
             CertificatePostApprovalViewModel certificateApprovalViewModel)
         {
+            var approvalsValidationFailed = await ValidateReasonForChange1(certificateApprovalViewModel);
+            if (approvalsValidationFailed != null)
+                return approvalsValidationFailed;
+
             certificateApprovalViewModel.UserName = ContextAccessor.HttpContext.User
                 .FindFirst("http://schemas.xmlsoap.org/ws/2005/05/identity/claims/upn")?.Value;
 
-            ApiClient.ApproveCertificates(certificateApprovalViewModel);
+            SetPrivatelyFundedCertApprovalStatus(certificateApprovalViewModel);
+            DeterminNextActionUsingCurrrentStatus(certificateApprovalViewModel);
+
+            await ApiClient.ApproveCertificates(certificateApprovalViewModel);
+            return RedirectToAction(certificateApprovalViewModel.ActionHint);
+        }
+        
+        [HttpGet]
+        public async Task<FileContentResult> ExportSentForApproval()
+        {
+            var certificates = await ApiClient.GetCertificatesToBeApproved(0,1, CertificateStatus.ToBeApproved, null);
+            var data = certificates?.Items
+                .Select(x => ToDictionary<object>(x, ApprovalToExcelAttributeMappings()));
+          
+            using (var package = new ExcelPackage())
+            {
+                var worksheet = package.Workbook.Worksheets.Add("Sheet1");
+                worksheet.Cells.LoadFromDataTable(ToDataTable(data), true);
+
+                return File(package.GetAsByteArray(), "application/excel", $"CerificatesToBeApproved-{DateTime.Now:dd-MM-yyyyTHH:mm:ss}.xlsx");
```

Comment:
Typo fixed

Reply:
I spoke to alan regarding the export of data and since this will only be accessible after you login and will be used in back office, it is not a issue when it comes to security.